crates/orderbook/src/api/post_quote.rs

```diff
@@ -39,6 +39,8 @@ pub struct OrderQuoteRequest {
     buy_token_balance: BuyTokenDestination,
     #[serde(default)]
     price_quality: PriceQuality,
+    #[serde(default)]
+    is_liquidity_order: bool,
```

Comment:
Make `is_liquidity_order` optional with `false` as default to minimize breaking backwards compatibility.